### ISSUE TITLE
yaml package specifies invalid commit in glide.lock

### ISSUE THREAD

**Issue by brianredbeard** [coreos/mayday#34]
While performing:

```
glide install --update-vendored

[INFO]	--> Setting version for gopkg.in/yaml.v2 to 80ffc689d4f094f4f1ffa3409332aa3d87e87c7a.
[ERROR]	Failed to set version on gopkg.in/yaml.v2 to 80ffc689d4f094f4f1ffa3409332aa3d87e87c7a: Unable to update checked out version
```

Troubleshooting this further it seems that [`80ffc689d4f094f4f1ffa3409332aa3d87e87c7a`](https://github.com/go-yaml/yaml/commit/80ffc689d4f094f4f1ffa3409332aa3d87e87c7a) is an invalid commit hash.

Updating `glide.lock` to a new commit works as expected:
```
- name: gopkg.in/yaml.v2
  version: e4d366fc3c7938e2958e662b4258c7a89e1f0e3e
```

```
[INFO]	--> Setting version for gopkg.in/yaml.v2 to e4d366fc3c7938e2958e662b4258c7a89e1f0e3e.
...
[INFO]	Cleaning up vendored package gopkg.in/yaml.v2
```

**Comment by tschuy** [comment]
That's certainly weird.

I've updated the `glide.lock` file in #41 and installed the . It appears to work fine ¯\\\_(ツ)\_/¯

**Comment by tschuy** [comment]
Included in #43
